integration/fixtures/fake-client-secrets-3lo-loopback.json

```diff
@@ -2,7 +2,7 @@
   "installed": {
     "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
     "auth_uri": "https://accounts.google.com/o/oauth2/auth",
-    "client_email": "",
+    "client_email": "force-cache-diff",
```

Comment:
FYI: this is necessary when running commands that use the cache. For example, tests in the original 3LO that do not bypass the cache will create a new cache entry. When tests for the loopback version run without bypassing the cache will return the previously generated cache entry - the same cache entry generated by the out of band version. This is because the redirect_uri is completely removed when creating a cache entry. In other words, redirect_uris={A,B} == redirect_uris={B,A}.

Reply:
Introduced new email addresses as previously discussed.